### PR TITLE
Update tokens

### PR DESCRIPTION
**Pull request by ocefpaf**
Address https://github.com/conda-forge/pandas-feedstock/issues/56#issuecomment-454019965

Ping @TomAugspurger

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -21,7 +20,7 @@ requirements:
     - python
     - pip
     - cython
-    - numpy 1.12
```
Comment (TomAugspurger) [inline recipe/meta.yaml]: Should the `host` NumPy be dependent on the python version, so that we're building against the oldest possible NumPy ABI?

Comment (ocefpaf) [reply recipe/meta.yaml]: We don't have numpy 1.12 built for Python 3.7. I was lazy here and bumped it to 1.14 but ideally we should build a 1.12 for Python 3.7.

Comment (TomAugspurger) [reply recipe/meta.yaml]: Indeed. I'm happy with whatever gets this working for the RC :)

**Comment by TomAugspurger** [comment]
Thanks @ocefpaf. Do you think this is good to go on your end?

**Comment by ocefpaf** [comment]
> Thanks @ocefpaf. Do you think this is good to go on your end?

Yep. However, this will still upload it to the main channel. I'm not sure what is going on with @CJ-Wright's example in https://github.com/conda-forge/pandas-feedstock/issues/56#issuecomment-454063988

In the past we could only edit the conda-forge.yml and re-render, now it seems that there are a few more steps involved.

**Comment by ocefpaf** [comment]
@TomAugspurger let's merge this one and I'll move the packages manually as soon as they are build. I'll try to fix the automatic upload to the right label later.

**Pull request merged**